<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>samar</class>
 <widget class="QDialog" name="samar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1310</width>
    <height>921</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>1051</width>
     <height>671</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
  background-color: #87CEEB; /* blue sky color */
  border: 2px solid gray;
  border-radius: 5px;
  padding: 5px;
}

QLabel {
  color: white;
  font-size: 14px;
}

QLineEdit {
  background-color: white;
  border: 2px solid gray;
  border-radius: 5px;
  padding: 2px;
}

QPushButton {
  background-color: white;
  border: 2px solid gray;
  border-radius: 5px;
  color: gray;
  padding: 5px;
  font-size: 14px;
}

QPushButton:hover {
  background-color: gray;
  color: white;
}</string>
   </property>
   <property name="title">
    <string>Gestion Client</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>931</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #87CEEB;

QGroupBox {
  background-color: #87CEEB; /* blue sky color */
  border: 2px solid gray;
  border-radius: 5px;
  padding: 5px;
}

QLabel {
  color: white;
  font-size: 14px;
}

QLineEdit {
  background-color: white;
  border: 2px solid gray;
  border-radius: 5px;
  padding: 2px;
}

QPushButton {
  background-color: white;
  border: 2px solid gray;
  border-radius: 5px;
  color: gray;
  padding: 5px;
  font-size: 14px;
}

QPushButton:hover {
  background-color: gray;
  color: white;
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>190</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
      color: rgb(251, 252, 2521);
      font: 10pt &quot;Verdana&quot;;
      border: None;
      border-bottom-color: white;
      border-radius: 10px;
      padding: 0 8px;
      background: rgb(251, 252, 252);
      selection-background-color: darkgray;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>220</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
      color: rgb(251, 252, 2521);
      font: 10pt &quot;Verdana&quot;;
      border: None;
      border-bottom-color: white;
      border-radius: 10px;
      padding: 0 8px;
      background: rgb(251, 252, 252);
      selection-background-color: darkgray;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>250</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
      color: rgb(251, 252, 2521);
      font: 10pt &quot;Verdana&quot;;
      border: None;
      border-bottom-color: white;
      border-radius: 10px;
      padding: 0 8px;
      background: rgb(251, 252, 252);
      selection-background-color: darkgray;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>280</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
      color: rgb(251, 252, 2521);
      font: 10pt &quot;Verdana&quot;;
      border: None;
      border-bottom-color: white;
      border-radius: 10px;
      padding: 0 8px;
      background: rgb(251, 252, 252);
      selection-background-color: darkgray;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>310</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
      color: rgb(251, 252, 2521);
      font: 10pt &quot;Verdana&quot;;
      border: None;
      border-bottom-color: white;
      border-radius: 10px;
      padding: 0 8px;
      background: rgb(251, 252, 252);
      selection-background-color: darkgray;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>340</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
      color: rgb(251, 252, 2521);
      font: 10pt &quot;Verdana&quot;;
      border: None;
      border-bottom-color: white;
      border-radius: 10px;
      padding: 0 8px;
      background: rgb(251, 252, 252);
      selection-background-color: darkgray;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>200</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>id</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>230</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>260</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>290</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Adresse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>320</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>340</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>numero</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>370</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>370</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>suivant</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>0</y>
        <width>151</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../Downloads/Downloads/333441755_1627823517733294_9150439995447617027_n.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>-110</x>
        <y>-30</y>
        <width>1221</width>
        <height>431</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../Downloads/Downloads/332319502_992673331694691_7199529608011094507_n.jpg</pixmap>
      </property>
     </widget>
     <zorder>label_20</zorder>
     <zorder>lineEdit</zorder>
     <zorder>lineEdit_2</zorder>
     <zorder>lineEdit_3</zorder>
     <zorder>lineEdit_4</zorder>
     <zorder>lineEdit_5</zorder>
     <zorder>lineEdit_6</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
     <zorder>label_5</zorder>
     <zorder>label_6</zorder>
     <zorder>pushButton</zorder>
     <zorder>pushButton_3</zorder>
     <zorder>label_8</zorder>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>20</y>
        <width>341</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_9">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>200</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
      color: rgb(251, 252, 2521);
      font: 10pt &quot;Verdana&quot;;
      border: None;
      border-bottom-color: white;
      border-radius: 10px;
      padding: 0 8px;
      background: rgb(251, 252, 252);
      selection-background-color: darkgray;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_10">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>230</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
      color: rgb(251, 252, 2521);
      font: 10pt &quot;Verdana&quot;;
      border: None;
      border-bottom-color: white;
      border-radius: 10px;
      padding: 0 8px;
      background: rgb(251, 252, 252);
      selection-background-color: darkgray;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_11">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>260</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
      color: rgb(251, 252, 2521);
      font: 10pt &quot;Verdana&quot;;
      border: None;
      border-bottom-color: white;
      border-radius: 10px;
      padding: 0 8px;
      background: rgb(251, 252, 252);
      selection-background-color: darkgray;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_12">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>290</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
      color: rgb(251, 252, 2521);
      font: 10pt &quot;Verdana&quot;;
      border: None;
      border-bottom-color: white;
      border-radius: 10px;
      padding: 0 8px;
      background: rgb(251, 252, 252);
      selection-background-color: darkgray;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_13">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>320</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
      color: rgb(251, 252, 2521);
      font: 10pt &quot;Verdana&quot;;
      border: None;
      border-bottom-color: white;
      border-radius: 10px;
      padding: 0 8px;
      background: rgb(251, 252, 252);
      selection-background-color: darkgray;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_14">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>350</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
      color: rgb(251, 252, 2521);
      font: 10pt &quot;Verdana&quot;;
      border: None;
      border-bottom-color: white;
      border-radius: 10px;
      padding: 0 8px;
      background: rgb(251, 252, 252);
      selection-background-color: darkgray;
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>prenom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>adresse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>email</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>num</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>360</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>fidelite</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>380</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>suivant</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>380</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
      color: rgb(251, 252, 2521);
      font: 10pt &quot;Verdana&quot;;
      border: None;
      border-bottom-color: white;
      border-radius: 10px;
      padding: 0 8px;
      background: rgb(251, 252, 252);
      selection-background-color: darkgray;
}</string>
      </property>
      <property name="text">
       <string>précédent</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>380</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
      color: rgb(251, 252, 2521);
      font: 10pt &quot;Verdana&quot;;
      border: None;
      border-bottom-color: white;
      border-radius: 10px;
      padding: 0 8px;
      background: rgb(251, 252, 252);
      selection-background-color: darkgray;
}</string>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>-500</x>
        <y>-50</y>
        <width>1321</width>
        <height>491</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../Downloads/Downloads/332319502_992673331694691_7199529608011094507_n.jpg</pixmap>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>RECHERCHER</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>9</y>
        <width>151</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>TRIER</string>
      </property>
      <widget class="QPushButton" name="pushButton_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>trier</string>
       </property>
      </widget>
     </widget>
     <zorder>label_18</zorder>
     <zorder>tableView</zorder>
     <zorder>lineEdit_9</zorder>
     <zorder>lineEdit_10</zorder>
     <zorder>lineEdit_11</zorder>
     <zorder>lineEdit_12</zorder>
     <zorder>lineEdit_13</zorder>
     <zorder>lineEdit_14</zorder>
     <zorder>label_9</zorder>
     <zorder>label_10</zorder>
     <zorder>label_11</zorder>
     <zorder>label_12</zorder>
     <zorder>label_13</zorder>
     <zorder>label_14</zorder>
     <zorder>pushButton_4</zorder>
     <zorder>pushButton_6</zorder>
     <zorder>pushButton_8</zorder>
     <zorder>groupBox_2</zorder>
     <zorder>groupBox_3</zorder>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>360</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_15">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>360</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
      color: rgb(251, 252, 2521);
      font: 10pt &quot;Verdana&quot;;
      border: None;
      border-bottom-color: white;
      border-radius: 10px;
      padding: 0 8px;
      background: rgb(251, 252, 252);
      selection-background-color: darkgray;
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>390</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>suivant</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>390</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>précédent</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>360</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>id</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>-330</x>
        <y>0</y>
        <width>1181</width>
        <height>621</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../Downloads/Downloads/332319502_992673331694691_7199529608011094507_n.jpg</pixmap>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>391</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>340</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>PDF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>80</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>FAIRE LE STATISTIQUE</string>
      </property>
     </widget>
     <zorder>label_19</zorder>
     <zorder>pushButton_2</zorder>
     <zorder>lineEdit_15</zorder>
     <zorder>pushButton_5</zorder>
     <zorder>pushButton_7</zorder>
     <zorder>label_16</zorder>
     <zorder>textBrowser</zorder>
     <zorder>pushButton_10</zorder>
     <zorder>pushButton_11</zorder>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QWidget" name="stars" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>421</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="captchavalue">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>401</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>240</y>
        <width>401</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>valider</string>
      </property>
     </widget>
     <widget class="QLabel" name="captchaLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>411</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_14">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>360</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>précédente</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>381</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Menu</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
